compaction/compaction.cc

```diff
@@ -1768,18 +1817,25 @@ static future<compaction_result> scrub_sstables_validate_mode(sstables::compacti
     };
 }
 
+future<compaction_result> scrub_sstables_validate_mode(sstables::compaction_descriptor descriptor, compaction_data& cdata, table_state& table_s, compaction_read_monitor_generator_wrapper& monitor_generator = default_read_monitor_wrapper()) {
+    monitor_generator.set_generator(std::make_unique<compaction_read_monitor_generator>(table_s, false));
+    auto res = co_await scrub_sstables_validate_mode(descriptor, cdata, table_s, *monitor_generator._generator);
```

Comment:
please replace boolean by bool_class use_backlog_tracker, so the intention is known at call site.